src/utils/detect-functions-builder.js

```diff
@@ -7,6 +7,7 @@ module.exports.detectFunctionsBuilder = async function (projectDir) {
     .map((det) => require(path.join(__dirname, '..', `function-builder-detectors/${det}`)))
 
   for (const detector of detectors) {
+    // eslint-disable-next-line no-await-in-loop
```

Comment:
At the moment, there is only one detector. In the future, we'll want to parallelize.